### ISSUE TITLE
Add 'cen' to the SPEC-like API.

### ISSUE THREAD

**Issue by danielballan** [danielballan/bluesky#140]
`cen` does not seem to be a "core" SPEC macro, so there is no canonical implementation to mimic. It looks like a typical implmentation moves the motors to a position defined by some global constant, `CENTER`. Easy enough.

But, going further, we build this on more sophisticated functionality. What if Movable objects had an optional, mutable `home` attribute, a dict? Then, a "reset beamline" function could call `movable.set(movable.home)` on every `movable` we know about?

**Comment by danielballan** [comment]
Per @cmazzoli, we could call this "park," as in putting a car into park.

**Comment by cmazzoli** [comment]
Hi Dan,
thank you and sorry for my delay.

cen was indeed a shortcut added because the global in SPEC had an awful name both to remember and to write.
About "park": it sounds a nice and useful feature. Sometimes you need to keep into account the interaction with other motors / positions / part of the beamline and or the experiment and might be quite complex. Happy to discuss with you whenever you like to.

Ciao!

**Comment by danielballan** [comment]
This should be implemented as an example in the documentation using `BestEffortCallback.peaks`, `bluesky.plans.mv`, and some local settings.

**Comment by ambarb** [comment]
@danielballan I see this isn't closed.  Does that mean we can get it?

**Comment by danielballan** [comment]
Yes.

**Comment by danielballan** [comment]
Revisiting, we're not sure whether to close it. @ambarb @cmazzoli Do you have a custom plan that serves this need now? Is it worth generalizing for bluesky?

**Comment by stan-dot** [comment]
@coretl  is this a scanspec use case?

**Comment by coretl** [comment]
> @coretl is this a scanspec use case?

No, scanspec defines where to go during a scan, this looks like it is about moving motors to a static fixed "park" position so they are in a known good or out of the way position before other motors are used for a scan

**Comment by stan-dot** [comment]
@callumforrester  are we interested in this 'set to known' functionality? to be honest this sounds similar to the lookup table discussion

https://github.com/DiamondLightSource/blueapi/issues/537

**Comment by callumforrester** [comment]
I have no concrete use cases at present
